src/workerd/api/data-url.c++

```diff
@@ -10,6 +10,18 @@ kj::Maybe<DataUrl> DataUrl::tryParse(kj::StringPtr url) {
   return kj::none;
 }
 
+constexpr kj::FixedArray<uint8_t, 256> ascii_whitespace_table = []() consteval {
```

Comment:
Can this be static? Not sure if it makes much of a difference with constexpr. Same goes for mimetype.c++ `token_table` and `quoted_string_token_table`.

Reply:
Yeah, no worries!